src/routes/Routes.js

```diff
@@ -1,7 +1,7 @@
-var BASE_URL = process.env.REACT_APP_BACKEND_URL || 'https://spica.eic.cefet-rj.br';
+var BASE_URL = process.env.REACT_APP_BACKEND_URL || 'http://localhost:8000';
```

Comment:
We cant commit localhost as BASE_URL to main because this code is used in production environment spica